aries_vcx/src/common/ledger/transactions.rs

```diff
@@ -124,6 +125,46 @@ pub async fn into_did_doc(profile: &Arc<dyn Profile>, invitation: &Invitation) -
     Ok(did_doc)
 }
 
+fn _ed25519_public_key_to_did_key(public_key_base58: &str) -> VcxResult<String> {
+    let public_key_bytes = bs58::decode(public_key_base58).into_vec().unwrap();
```

Comment:
preferably no `unwrap` as this could panic the program. should throw an error similar to my comment below